contracts/FluxPriceFeedFactory.sol

```diff
@@ -0,0 +1,134 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.12;
+
+import "@openzeppelin/contracts/access/AccessControl.sol";
+import "@openzeppelin/contracts/utils/Strings.sol";
+import "./interface/IERC2362.sol";
+import "./FluxPriceFeed.sol";
+import "hardhat/console.sol";
```

Comment:
I think we should use this only for debugging. :)

Some `console.log` should be removed.